libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/ProgressDialog.kt

```diff
@@ -61,12 +62,12 @@ fun ProgressDialog(
     }
     Dialog(
         onDismissRequest = onDismissRequest,
-        properties = DialogProperties(dismissOnBackPress = false, dismissOnClickOutside = false)
+        properties = properties,
     ) {
         ProgressDialogContent(
             modifier = modifier,
             text = text,
-            isCancellable = isCancellable,
+            isCancellable = showCancelButton,
```

Comment:
Also rename `ProgressDialogContent.isCancellable` parameter to `ProgressDialogContent.showCancelButton`?